src/daft-physical-plan/src/physical_planner/translate.rs

```diff
@@ -547,7 +547,10 @@ pub fn adaptively_translate_single_logical_node(
 pub fn extract_agg_expr(expr: &ExprRef) -> DaftResult<AggExpr> {
     match expr.as_ref() {
         Expr::Agg(agg_expr) => Ok(agg_expr.clone()),
-        Expr::Window(inner_expr, _) => extract_agg_expr(inner_expr),
+        Expr::Over(inner_expr, _) => {
+            let expr_ref: ExprRef = inner_expr.into();
+            extract_agg_expr(&expr_ref)
+        }
```

Comment:
Didn't catch this in a previous PR but I don't think this should be done actually. I don't think we should ever try to extract agg expressions from Over directly, we should first get the window expression out of over and check if it's an agg, which we would probably want to do elsewhere, such as in the physical plan translation to determine what operator to use.